index.html

```diff
@@ -306,26 +310,28 @@ <h2 class="inter">
 					<textarea  class="box" name="msg" maxlength="500" placeholder="Write me something..." required></textarea>
 					<button type="submit" class="btn">Get in touch</button>
 				</form>
-				<hr>
-				<ul class="list-icons dark">
-					<li>
-						<a href="#"><img src="Works-Images/Github dark.png"></a>
-					</li>
-					<li>
-						<a href="#"><img src="Works-Images/Linkedin dark.png"></a>
-					</li>
-					<li>
-						<a href="#"><img src="Works-Images/Peace dark.png"></a>
-					</li>
-					<li>
-						<a href="#"><img src="Works-Images/Twitter dark.png"></a>
-					</li>
-					<li>
-						<a href="#"><img src="Works-Images/M dark.png"></a>
-					</li>
-				</ul>
 			</section>
 		</section>
+		<footer>
+			<hr>
+			<ul class="list-icons dark">
+				<li>
+					<a href="#"><img src="Works-Images/Github dark.png"></a>
+				</li>
+				<li>
+					<a href="#"><img src="Works-Images/Linkedin dark.png"></a>
+				</li>
+				<li>
+					<a href="#"><img src="Works-Images/Peace dark.png"></a>
+				</li>
+				<li>
+					<a href="#"><img src="Works-Images/Twitter dark.png"></a>
+				</li>
+				<li>
+					<a href="#"><img src="Works-Images/M dark.png"></a>
+				</li>
+			</ul>
```

Comment:
Please stick to the given Figma design as much as possible according to the project requirement. Currently, the size of your social media icon and the space between your social media icon are large. Kindly, fix it.
![image](https://github.com/hamidazim321/Portfolio-Project/assets/101523582/a5af46f2-19fe-4b03-b3a0-a3e029a21187)

Your design

![image](https://github.com/hamidazim321/Portfolio-Project/assets/101523582/a4dca322-9b24-4e94-b09a-5e09f4577f27)

Figma design

![image](https://github.com/hamidazim321/Portfolio-Project/assets/101523582/0de1713f-d78e-47f4-9808-bb4504472065)